README.md

```diff
@@ -17,28 +17,55 @@ Fourier Accelerated Nodal Solvers (FANS) is an FFT-based homogenization solver d
 
 Before proceeding with the installation, ensure that your system has the necessary dependencies. The prerequisites of FANS can be installed using Spack for a streamlined setup on high-performance computing systems, or through traditional package management for personal use.
 
-### Traditional Installation
-
-If you're setting up FANS on a personal computer or in a non-HPC environment, follow these instructions:
-
-Please ensure you have the following dependencies installed on your system:
+The requirements are:
 
-- CMake (version 3.0 or higher)
+- A C++ compiler (e.g. GCC)
+- CMake (version 3.0 or higher) (+ GNU file utility for creating .deb packages)
+- Git (for cloning this repo)
 - MPI (mpicc and mpic++)
 - HDF5 with parallel support
 - Eigen3
 - FFTW3 with MPI support
 
-Specifically, to run FANS, you at least need the following packages:
+### Traditional Installation
+
+If you're setting up FANS on a personal computer or in a non-HPC environment, please use the apt package manager. For the build toolchain:
+
+```bash
+apt-get install \
+    software-properties-common \
+    build-essential \
+    cmake \
+    file \
+    git
+```
+
+For the FANS dependencies:
 
 ```bash
-openmpi-bin libc6 libfftw3-double3 libfftw3-mpi3 libgcc-s1 libgomp1 libhdf5-103 libopenmpi3 libstdc++6
+apt-get install \
+    libhdf5-dev \
+    libopenmpi-dev \
+    libeigen3-dev \
+    libfftw3-dev \
+    libfftw3-mpi-dev
 ```
 
-To build fans, you additionally need these packages:
+Also we recommend to install these commonly used tools with FANS and setup a virtualenv for the `h52xdmf.py` script:
```

Comment:
```suggestion
Also, we recommend to install these commonly used tools with FANS and setup a virtualenv for the `h52xdmf.py` script:
```
Some more information on why we ask users to do this would be helpful here. It is not very clear on why one needs this.

Reply:
okay, it's not only for the `h52xdmf.py` script... it's for the whole FANS_Dashboard which is python (and Jupyter notebook) based

Reply:
I would writing separate documentation for the dashboard, and not mix things with FANS building and installation.